src/test/resources/contracts/offender-bookings.avsc

```diff
@@ -0,0 +1,358 @@
+    },
+    {
+      "name": "OFFENDER_ID",
+      "type": "long",
+      "nullable": true,
+      "glossary-index": "offender:id",
+      "description": "The unique identifier for an offender."
+    },
+    {
+      "name": "AGY_LOC_ID",
+      "type": "string",
+      "nullable": true,
+      "glossary-index": "location:id",
+      "description": "The location residing within an agency."
+    },
+    {
+      "name": "LIVING_UNIT_ID",
+      "type": "long",
+      "nullable": true,
+      "glossary-index": "living_unit:id",
+      "description": "System generated id for offender bed location."
+    },
+    {
+      "name": "DISCLOSURE_FLAG",
+      "type": "string",
+      "nullable": false,
+      "description": "A flag relating to the disclosure of offender information. Default value 'Y'"
+    },
+    {
+      "name": "IN_OUT_STATUS",
+      "type": "string",
+      "nullable": false,
+      "description": "Reference Code ( IN_OUT_STS ) Indicates whether the offender is currently in or out of the facility."
+    },
+    {
+      "name": "ACTIVE_FLAG",
+      "type": "string",
+      "nullable": false,
+      "description": "Indicates whether the offender is active or inactive at the location. Default value 'N'"
+    },
+    {
+      "name": "BOOKING_STATUS",
+      "type": "string",
+      "nullable": true,
+      "description": "Reference Code ( BOOK_STS ). The open or closed status of the booking."
+    },
+    {
+      "name": "YOUTH_ADULT_CODE",
+      "type": "string",
+      "nullable": true,
+      "description": "Reference Code ( YOUTH_ADULT ) : Indicates whether the offender is currently a youth or adult."
+    },
+    {
+      "name": "FINGER_PRINTED_STAFF_ID",
+      "type": "long",
+      "nullable": true,
+      "description": "The staff member finger printing the offender."
+    },
+    {
+      "name": "SEARCH_STAFF_ID",
+      "type": "long",
+      "nullable": false,
+      "description": "Id of staff member who searched offender."
+    },
+    {
+      "name": "PHOTO_TAKING_STAFF_ID",
+      "type": "long",
+      "nullable": false,
+      "description": "Id of staff member who took offender's image"
+    },
+    {
+      "name": "ASSIGNED_STAFF_ID",
+      "type": "long",
+      "nullable": true,
+      "description": "COMM Side assigned Staff ID."
+    },
+    {
+      "name": "CREATE_AGY_LOC_ID",
+      "type": "string",
+      "nullable": true
+    },
+    {
+      "name": "BOOKING_TYPE",
+      "type": "string",
+      "nullable": true
+    },
+    {
+      "name": "BOOKING_CREATED_DATE",
+      "type": {
+        "type": "int",
+        "logicalType": "date"
+      },
+      "nullable": false
+    },
+    {
+      "name": "ROOT_OFFENDER_ID",
+      "type": "long",
+      "nullable": true,
+      "glossary-index": "offender:id",
+      "description": "The Root Offender Identifier"
+    },
+    {
+      "name": "AGENCY_IML_ID",
+      "type": "long",
+      "nullable": true
+    },
```

Comment:
should be a glossary index foreign key to internal-locations:id or living-unit_id (whichever is decided)

Reply:
checked with Zahoor and he confirmed glossary-index is not part of this ticket and can be handled in another backlog ticket he is working on